plugins/node-logs.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: node-logs
+spec:
+  version: v0.0.1
+  homepage: https://github.com/aravindhp/kubectl-node-logs
+  shortDescription: Plugin for viewing and filtering node service logs
```

```suggestion
  shortDescription: View and filter node service logs
```
It's clear that it's a plugin

---

That makes sense. I have made the change. Please merge when you get the chance.